### ISSUE TITLE
Make Action Workflow accessibility friendly

### ISSUE THREAD

**Issue by rizaardiyanto1412** [publishpress/PublishPress-Future#886]
I am a blind person, so I use a screen reader (JAWS) to manage a WordPress site where I am the administrator.
The PublishPress Future interface for Action Workflows does not work with a screen reader (keyboard navigation only).
Could you please make the necessary modifications to make the PublishPress products interface compatible with a screen reader?
Thank you for your help.

Original request: https://secure.helpscout.net/conversation/2666650408/224721?viewId=6932143

**Comment by andergmartins** [comment]
We got another request for this to add screen reader support to the workflow builder. The user can't use it; he can only use the legacy Future Actions.
